tests/python/tir-transform/test_tir_transform_lower_tvm_builtin.py

```diff
@@ -260,11 +260,13 @@ def expected():
 
 
 class TestLowerAllocateRequiresDeviceID(tvm.testing.CompareBeforeAfter):
+    """If device id is missing, error."""
+
     transform = tvm.tir.transform.LowerTVMBuiltin()
 
     def before():
         T.func_attr({"target": T.target("llvm")})
-        T.attr("dummy", "device_id", 0)
+        T.attr("dummy", "device_type", 2)  # kDLCuda
```

Comment:
Is this constant defined anywhere on the Python side? Perhaps it should be, to avoid such magic numbers (if we expect that to happen again).

Reply:
Good question, and looks like it is defined in the `tvm.runtime.Device` struct.  I've updated the usage here, and throughout this unit test file.